src/test/scala/edu/ie3/simona/agent/grid/congestion/VoltageRangeSpec.scala

```diff
@@ -243,103 +207,103 @@ class VoltageRangeSpec
         VoltageLimits(0.9, 1.1),
         gridComponents,
         Map(
-          inferior1.ref -> (VoltageRange(0.1.asPu, 0.01.asPu), Set(
+          inferior1.ref -> (VoltageRange(0.1, 0.01), Set(
             tappingModel
           )),
-          inferior2.ref -> (VoltageRange(0.01.asPu, -0.04.asPu), Set(
+          inferior2.ref -> (VoltageRange(0.01, -0.04), Set(
             tappingModel
           )),
         ),
         subnetNo = 1,
       )
 
-      range.deltaPlus should equalWithTolerance(0.04.asPu)
-      range.deltaMinus should equalWithTolerance(-0.02.asPu)
-      range.suggestion should equalWithTolerance(0.asPu)
+      range.deltaPlus should approximate(Each(-0.03))
+      range.deltaMinus should approximate(Each(-0.02))
+      range.suggestion should approximate(Each(-0.02))
```

Comment:
Could you just give me some short update why these values changed (also others here)